<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>571</width>
      <height>341</height>
     </rect>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>258</width>
       <height>292</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QSlider" name="speedSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="1">
         <widget class="QPushButton" name="backwardButton">
          <property name="text">
           <string>Backward</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="forwardButton">
          <property name="text">
           <string>Forward</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="rightButton">
          <property name="text">
           <string>Right</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="leftButton">
          <property name="text">
           <string>Left</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="cameraLabel">
        <property name="text">
         <string>Camera Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="accelerometerLabel">
        <property name="text">
         <string>Accelerometer Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="batteryLabel">
        <property name="text">
         <string>Battery Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="imuLabel">
        <property name="text">
         <string>IMU Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ultrasonicLabel">
        <property name="text">
         <string>Ultraonic Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="gpsLabel">
        <property name="text">
         <string>GPS Data</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
